docs/versioning.rst

```diff
@@ -0,0 +1,156 @@
+**********
+Versioning
+**********
+
+
+Current Release Stages
+======================
+
+.. list-table:: Releases
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Release
+     - Runtimes
+     - Minimum Trace Agent
+     - Stage
+   * - 0.x
+     - CPython: 2.7, 3.5–3.10
+     - 
+     - MAINTENANCE [2022-03-01]
+   * - 1.x
+     - CPython: 2.7, 3.5+
+     - 7.28
+     - CURRENT
+
+
+Runtimes
+========
+
+
+.. list-table:: Supported runtime versions
+   :widths: 60 40
+   :header-rows: 1
+
+   * - Runtime
+     - Runtime version
+   * - CPython 
+     - 3.10
+   * -
+     - 3.9
+   * -
+     - 3.8
+   * -
+     - 3.7
+   * -
+     - 3.6
+   * -
+     - 3.5
+   * -
+     - 2.7
+
+   
+.. list-table:: Supported operating systems
```

Comment:
I wonder if we need to merge this + last tables

if we ever add support for pypy maybe we only want to support linux x86-64?

would we ever want to do that?